server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/CliCFAST.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.cli.command;
+
+import com.google.common.io.Files;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import org.eclipse.lsp.cobol.cfg.CFASTBuilder;
+import org.eclipse.lsp.cobol.cli.di.CliModule;
+import org.eclipse.lsp.cobol.common.dialects.CobolLanguageId;
+import org.eclipse.lsp.cobol.common.model.tree.Node;
+import org.eclipse.lsp.cobol.common.pipeline.StageResult;
+import org.eclipse.lsp.cobol.dialects.ibm.ProcessingResult;
+import picocli.CommandLine;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.concurrent.Callable;
+
+/**
+ * Generates CFAST representation of the COBOL program in the defined folder
+ */
+@CommandLine.Command(name = "cfast", description = "generate cfast from cobol source")
+public class CliCFAST  implements Callable<Integer> {
+
+  @CommandLine.ParentCommand
+  private Cli parent;
+
+  @CommandLine.Option(
+      description = "Path to workspace folder.",
```

Comment:
The naming is a bit confusing. Other commands assume that workspace is defined by processor groups configuration and related configuration, like set of programs, copybooks path, and  copybook extensions is taken from processor group configuration.
So let's rename the option to something like ccf-tests-folder, or change logic so it use processor groups.

Reply:
It is a copy/paste from a different command. I am just following code style and known name conventions

Reply:
This command is not CCF related thing, it is for generating CFAST. So I have renamed it to the source_folder